### PR TITLE
Use new version of makefile which works on mac

### PR DESCRIPTION
**Pull request by nottrobin**
The accompaniment to this PR: https://github.com/ubuntudesign/docker-devrun/pull/1.

QA
--

``` bash
make check-for-docker
```

Check this ^ exits without a hitch.

``` bash
make run
```

Check this still runs the site.

### CODE REVIEW

**Comment by pmahnke** [comment]
the command check for docker worked... but the make run failed

➜  developer.ubuntu.com git:(master) make check-for-docker
➜  developer.ubuntu.com git:(fix-devrun-for-mac) make run

===
The webservice will be available at:
http://localhost:8017
===

Traceback (most recent call last):
  File "manage.py", line 8, in <module>
    from django.core.management import execute_from_command_line
ImportError: No module named django.core.management
Traceback (most recent call last):
  File "manage.py", line 8, in <module>
    from django.core.management import execute_from_command_line
ImportError: No module named django.core.management
Stopping a33da7d6_db_1 ... done

**Comment by pmahnke** [comment]
Works for me 👍

File: templates/includes/base.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html>
+<head>
+
+<title>{% block title %}{% endblock %} | Developer</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://assets.ubuntu.com/v1/docs-vanilla-theme-version-1.1.0.min.css" />
+<link rel="stylesheet" href="css/global.css" />
```
Comment (nottrobin) [inline templates/includes/base.html]: This line doesn't work. I think maybe you want to remove it entirely? Fixing it to correctly link to `/static/css/global.css` makes the page look worse in my opinion, but I don't know what @grahambancroft's intention was or what the designs say.

File: templates/includes/base.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html>
+<head>
+
+<title>{% block title %}{% endblock %} | Developer</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://assets.ubuntu.com/v1/docs-vanilla-theme-version-1.1.0.min.css" />
```
Comment (nottrobin) [inline templates/includes/base.html]: Obviously it's not ideal to be linking to docs-vanilla-theme from the assets server - we should be building it locally. But I guess it's fine for now as long as we know it's going to be removed later.

**Pull request merged**